langchain4j-chatglm/src/main/java/dev/langchain4j/model/chatglm/ChatGLMChatModel.java

```diff
@@ -0,0 +1,71 @@
+package dev.langchain4j.model.chatglm;
+
+import dev.langchain4j.data.message.AiMessage;
+import dev.langchain4j.data.message.ChatMessage;
+import dev.langchain4j.model.chat.ChatLanguageModel;
+import dev.langchain4j.model.output.Response;
+import lombok.Builder;
+
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static dev.langchain4j.internal.RetryUtils.withRetry;
+import static dev.langchain4j.internal.Utils.getOrDefault;
+
+public class ChatGLMChatModel implements ChatLanguageModel {
```

Comment:
please rename `...GLM...` into `...Glm...` in class names everywhere

Reply:
done